packages/ember/tests/dummy/app/templates/docs/coordination.md

```diff
@@ -0,0 +1,68 @@
+
+However, if you're dealing with hairy concurrency situations with multiple different logical threads of execution under test, understanding how different sets of milestones interact is key to ensuring you have the degreee of control you need and that your tests are deterministic.
+
+## Milestone Coordinators
+
+Each time you call `activateMilestones`, it creates a {{#link-to 'docs.api.item' 'modules/@milestones/core~MilestoneCoordinator'}}`MilestoneCoordinator`{{/link-to}} instance.
+
+When a milestone is reached, it checks to see if there's an active coordinator for either its ID or one of its tags. If so, it pauses until that coordinator tells it how to continue. When you call `advanceTo` with a milestone key, it locates the corresponding coordinator and waits until it's ready with the pending milestone, and then passes your instruction back to the paused milestone for how to proceed.
+
+A given milestone coordinator can only be paused at or advancing to a single milestone at a time. As mentioned in {{#link-to 'docs.interacting-with-milestones'}}Interacting with Milestones{{/link-to}}, if you're currently advancing to a particular milestone, all others belonging to the same coordinator that are reached along the way will automatically behave as if you called `.continue()` on them. This prevents a deadlock scenario where your tests are waiting to reach one milestone, but your application code is waiting to hear how to proceed with an earlier one.
+
+### Logical Threads of Execution
+
+While it's possible to have true parallelism with tools like [web workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API) in the browser or [worker threads](https://nodejs.org/api/worker_threads.html#worker_threads_worker_threads) in Node.js, most day-to-day usage of JavaScript is single-threaded. However, concurrent _logical_ threads of execution can easily coexist (or else there wouldn't be much point to this library), interleaving any time the call stack clears or you `await` something.
+
+This means you can easily have more than one conceptual 'thing' happening at a time in your application code during testing, and you may want to work with milestones in more than one of those logical threads at a time.
+
+### Working with Multiple Coordinators
+
+As a general rule, you typically want one milestone coordinator for each logical thread of execution you're testing together. Put another way, **the milestones for each conceptual flow of control under test should be activated in a separate `activateMilestones` call**.
+
+```ts
+activateMilestones([MilestonesFor, ABackgroundPollingOperation]);
+activateMilestones([OtherMilestones, ForTimingGrowlStyle, UserNotifications]);
+```
+
+When you call the `advanceTo` function importable from `@milestones/core`, it will automatically delegate to the active coordinator for the key you give it.
```

Comment:
> importable from `@milestones/core`

Might want parens or mdashes around this, or just call it the "importable `advanceTo` function" since the other `advanceTo` hangs off of the MilestoneCoordinator.